javascript/index.js

```diff
@@ -32,14 +52,26 @@ function printMilliseconds() {
 
 function printSplit() {
   // ... your code goes here
+  var childElement = document.createElement("LI");
+  childElement.innerHTML = chronometer.splitClick()
+  splits.appendChild(childElement);
+
 }
 
 function clearSplits() {
   // ... your code goes here
+ 
+  while (splits.hasChildNodes()) {  
+    splits.removeChild(splits.firstChild);
+  }
+  
+  
 }
 
 function setStopBtn() {
   // ... your code goes here
+  chronometer.stopClick;
+
 }
 
```

Comment:
Similarly, functions setSplitBtn,setStartBtn and setResetBtn can also be used for handling DOM manipulation
```
function setSplitBtn() {
  btnRight.className = 'btn split';
  btnRight.textContent = 'SPLIT';
}

function setStartBtn() {
  btnLeft.className = 'btn start';
  btnLeft.textContent = 'START';
}

function setResetBtn() {
  btnRight.className = 'btn reset';
  btnRight.textContent = 'RESET';
}